appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js

```diff
@@ -253,13 +253,13 @@ export const getAddDocuments = async (request, response) => {
 	let uploadPageHeadingText = '';
 
 	switch (documentType) {
-		case 'siteNotice':
+		case 'whoNotifiedSiteNotice':
```

Comment:
These could be changed to the constants, i.e. `APPEAL_DOCUMENT_TYPE.WHO_NOTIFIED_SITE_NOTICE`...

Reply:
sorted. also for the `notificationDocumentTypes` array below